packages/botonic-cli/src/commands/deploy.ts

```diff
@@ -148,7 +147,7 @@ Uploading...
       }
       // Show the current bot in credentials at top of the list
       const first_id = this.botonicApiService.bot.id
-      bots.sort(function (x, y) {
+      bots.sort(function(x, y) {
```

Comment:
@marcrabat please ensure you have pre-commit installed and execute "npm i -D" to update to prettier 2

Reply:
fixed 👍